prime-router/src/main/kotlin/fhirengine/azure/FHIRFlowFunctions.kt

```diff
@@ -26,7 +31,10 @@ const val fhirQueueName = "fhir-raw-received"
 /**
  * Process will work through all the reports waiting in the 'process' queue
```

Comment:
Comment is incorrect and misleading, not updating after copying from the ProcessFunction

Reply:
Updating!